static/src/javascripts/projects/common/modules/commercial/adblock-messages.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import config from 'lib/config';
+import detect from 'lib/detect';
+import { local } from 'lib/storage';
+import userFeatures from 'commercial/modules/user-features';
+
+const adblockInUse = function() {
```

Comment:
Can we use arrow functions here? eg.

const adblockInUse = () => detect.adblockInUse;

This can be applied to the other functions in this module.